core/store/orm/orm.go

```diff
@@ -1314,16 +1318,20 @@ func (orm *ORM) DeleteFluxMonitorRoundsBackThrough(aggregator common.Address, ro
     `, aggregator, roundID).Error
 }
 
+// MostRecentFluxMonitorRoundID finds roundID of the most recent round that the provided oracle
+// address submitted to
 func (orm *ORM) MostRecentFluxMonitorRoundID(aggregator common.Address) (uint32, error) {
 	orm.MustEnsureAdvisoryLock()
 	var stats models.FluxMonitorRoundStats
-	err := orm.db.First(&stats, "aggregator = ?", aggregator).Order("round_id DESC").Error
+	err := orm.db.Order("round_id DESC").First(&stats, "aggregator = ?", aggregator).Error
```

Comment:
Easy and dangerous mistake to make this one, because it's silent. Chalk up one more reason for dumping gorm.